CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [unrealesed]
+
+### Added
+- Add user's Terms and Conditions status in the state
```

Comment:
- User's Terms and Conditions/Acceptable Use Policy (AUP) information in state information
- User's COPerson ID in state information

Reply:
Done